### DISCUSSION TITLE
Should reactions plugin exclude reactions from dependabot?

### DISCUSSION THREAD

**Discussion by spenserblack** [lowlighter/metrics#255] [Ideas]
Dependabot adds a :+1: reaction to command comments (e.g. `@dependabot merge`). This can weigh the statistics heavily if you use dependabot in a lot of repositories.

I think the purpose of the plugin is to show how useful your issue comments are to the community. Dependabot's reactions don't reflect how useful your comments are, since it's not a person reacting to them.

**Comment by lowlighter** [comment]
I will look into implementing an option to skip reactions of specified users, so dependabot and other bots account can be ignored 👍

**Reply by lowlighter** [reply]
This has been implemented in #256 and will be available on next version

**Reply by spenserblack** [reply]
Thank you!
